src/fparser/two/pattern_tools.py

```diff
@@ -287,6 +287,7 @@ def __call__(self, string):
     name = Pattern('<name>', r'[A-Z][\w$]*', flags=re.I)
 else:
     name = Pattern('<name>', r'[A-Z]\w*', flags=re.I)
+file_name = Pattern('<file_name>', '^(\S|\S.*\S)$', flags=re.I)
```

Comment:
Please add a comment to describe what this regex matches (I had to google it).

Reply:
A man of your skills, surely not. I have added a comment for mere mortals.

Reply:
Wow, that's a much more verbose comment than I was anticipating but thanks for adding!